src/lib/eslint/LWCEslintStrategy.ts

```diff
@@ -26,12 +21,16 @@ export class LWCEslintStrategy implements EslintStrategy {
 
 	private initialized: boolean;
 	protected logger: Logger;
+	private recommendedConfig: LooseObject;
 
 	async init(): Promise<void> {
 		if (this.initialized) {
 			return;
 		}
 		this.logger = await Logger.child(this.getEngine().valueOf());
+		const pathToRecommendedConfig = require.resolve('@salesforce/eslint-config-lwc')
+			.replace('index.js', 'base.js');
```

Comment:
@jfeingold35 Can you help me understand the reasoning behind this replacement?

Reply:
`index.js` is the file that gets pulled in by the base require() call. `base.js` is a file in the same directory that holds the ESLint config we're looking for.

Reply:
Thanks. What happens when either of these files doesn't exist in a future version? Would it move past gracefully?

Reply:
It's extremely unlikely that they'd rename their `index.js` file, because it's one of the standard names for the root file of NPM packages. And renaming the `base.js` file is the sort of change that would break the ruleset for everyone in the world, so they're unlikely to do that either.